src/components/TargetDetails/parseUtils.tsx

```diff
@@ -118,6 +202,30 @@ export const parseContent = (text: string | undefined) => {
   });
 };
 
+const grepTablesRegex = /(\|.+\|)/s;
+
+const parseTableContent = (text: string) => {
+  const parts = text.split(grepTablesRegex);
+
+  return parts.map(part => {
+    const tableMatched = part.match(grepTablesRegex);
+    if (tableMatched) {
+      return parseTables(part);
+    }
+
+    return parseNoneTableContent(part);
+  });
+};
+
+export const parseContent = (text: string) => {
+  const foundTables = text.match(grepTablesRegex);
```

Comment:
1. Check if the content contains a table, and then call the correct parsing function.